Modules/Loadable/VolumeRendering/qSlicerVolumeRenderingReader.cxx

```diff
@@ -80,13 +80,13 @@ vtkSlicerVolumeRenderingLogic* qSlicerVolumeRenderingReader::volumeRenderingLogi
 //-----------------------------------------------------------------------------
 QString qSlicerVolumeRenderingReader::description()const
 {
-  return "Transfer Function";
+  return tr("Transfer Function");
 }
 
 //-----------------------------------------------------------------------------
 qSlicerIO::IOFileType qSlicerVolumeRenderingReader::fileType()const
 {
-  return QString("TransferFunctionFile");
+  return tr(("TransferFunctionFile"));
```

Comment:
`TransferFunctionFile` is a token that must not be translated. It can be martked with `/*no tr*/` to make it clear that it was intentionally not translated.

```suggestion
  return /*no tr*/ QString("TransferFunctionFile");
```